src/BenchmarkDotNet/Templates/MonoAOTLLVMCsProj.txt

```diff
@@ -1,5 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk" DefaultTarget="Publish">
   <PropertyGroup>
+    <OriginalCSProjPath>$CSPROJPATH$</OriginalCSProjPath>
+    <MonoPropsPath>$([System.IO.Path]::ChangeExtension('$(OriginalCSProjPath)', '.Mono.props'))</MonoPropsPath>
+    <MonoTargetsPath>$([System.IO.Path]::ChangeExtension('$(OriginalCSProjPath)', '.Mono.targets'))</MonoTargetsPath>
```

Comment:
Does the `.Mono.targets` exist? I assume that this is added as a good practice.

Reply:
Yeah, that's a msbuild pattern of imports a props file before the project, and a targets file at the end. The naming is just a convention, and they are just normal msbuild files.
note: msbuild does support `Directory.Build.{props,targets}` which get imported automatically, but supporting that would need more work because these would need to get "deployed" accordingly for build of the generated project.